certora/confs/Marketplace.conf

```diff
@@ -1,18 +1,23 @@
 {
     "files": [
         "contracts/Marketplace.sol",
-        "certora/harness/MarketplaceHarness.sol",
-        "certora/helpers/ERC20A.sol"
+        "contracts/Groth16Verifier.sol",
+        //"certora/harness/MarketplaceHarness.sol",
+        "certora/helpers/ERC20A.sol",
     ],
-    "parametric_contracts": ["MarketplaceHarness"],
+    "parametric_contracts": ["Marketplace"],
     "link" : [
-        "Marketplace:_token=ERC20A"
+        "Marketplace:_token=ERC20A",
+        "Marketplace:_verifier=Groth16Verifier"
     ],
     "msg": "Verifying Marketplace",
     "rule_sanity": "basic",
     "verify": "Marketplace:certora/specs/Marketplace.spec",
     "optimistic_loop": true,
-    "loop_iter": "3"
+    "loop_iter": "3",
+    "solc": "solc8.23",
```

Comment:
This change here seems unnecessary. I don't think we want to force this binary name on others.

Reply:
`"solc": "solc8.23",` works on my laptop, that's why I'm using it. you can remove it, if you choose solc version in a different way